public/js/main.js

```diff
@@ -0,0 +1,45 @@
+//Animating chat bubbles on conversation cards:
+
+window.onload = function() {
+  //find all convo cards and loop through them
+  let convoCards = document.getElementsByClassName('conversation-card')
```

Comment:
I _always_ forget about `getElementsByClassName` and usually use `querySelector` and `querySelectorAll`. 👍